services/cluster-agent/cluster_agent/run.py

```
@@ -0,0 +1,191 @@
+    await ingest_client.close_client()
+    await prometheus.close_client()
+
+
+async def main() -> None:
+    logger.info("KubeX cluster agent starting (version %s)", AGENT_VERSION)
+    cluster_id = await bootstrap()
+
+    loop = asyncio.get_event_loop()
+    for sig in (signal.SIGINT, signal.SIGTERM):
+        try:
+            loop.add_signal_handler(sig, lambda: _shutdown_event.set())
```

Ruff (0.16.3)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)